packages/engine/src/scene/systems/ShadowSystem.tsx

```diff
@@ -226,11 +229,13 @@ const PlainCSMReactor = (props: { rendererEntity: Entity }) => {
 }
 
 const ChildCSMReactor = (props: { rendererEntity: Entity }) => {
-  const entities = useTreeQuery(props.rendererEntity)
+  const entities = useHookstate(useTreeQuery(props.rendererEntity))
```

Comment:
wrapping useHookstate around this is an anti pattern. the data already updates reactively